source/rsasign.c

```diff
@@ -46,36 +48,33 @@ static const char *tbs = "0123456789abcdefghij"; /* Length of SHA1 digest */
 
 static int threadcount;
 
-static OSSL_TIME *times = NULL;
-
 void do_rsasign(size_t num)
 {
     size_t i;
     unsigned char sig[64];
     EVP_PKEY_CTX *ctx = EVP_PKEY_CTX_new(rsakey, NULL);
     size_t siglen = sizeof(sig);
-    OSSL_TIME start, end;
+    OSSL_TIME time;
 
-    start = ossl_time_now();
+    counts[num] = 0;
 
-    for (i = 0; i < num_calls / threadcount; i++) {
+    do {
         if (EVP_PKEY_sign_init(ctx) <= 0
                 || EVP_PKEY_sign(ctx, sig, &siglen, tbs, SHA_DIGEST_LENGTH) <= 0) {
```

Comment:
To fix the warning when running make:
```suggestion
                || EVP_PKEY_sign(ctx, sig, &siglen, (const unsigned char *)tbs, SHA_DIGEST_LENGTH) <= 0) {
```